lib/Time/time.cpp

```diff
@@ -0,0 +1,38 @@
+#include "time.h"
+#include "Arduino.h"
+
+time::time()
+{
+    initateTimer2();
+    this->ticks = 0;
+}
+
+void time::initateTimer2()
+{
+    TCCR2A |= (1 << WGM21) | (0 << WGM20);
+    TCCR2B |= (0 << WGM22) | (1 << CS22) | (1 << CS21) | (1 << CS20);

```

Comment:
wgm is voor modes, cs is voor prescaler. misschien in aparte lines voor duidelijkheid